hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -713,9 +713,10 @@ public void stop() throws IOException {
       store.close();
       store = null;
     }
-    for (TableCacheMetrics metrics : tableCacheMetrics) {
+    tableCacheMetricsMap.entrySet().stream().forEach(metricsEntry -> {
+      TableCacheMetrics metrics = metricsEntry.getValue();
       metrics.unregister();
-    }
+    });
```

Comment:
This can be simplified:

```diff
-    tableCacheMetricsMap.entrySet().stream().forEach(metricsEntry -> {
-      TableCacheMetrics metrics = metricsEntry.getValue();
-      metrics.unregister();
-    });
+    tableCacheMetricsMap.values().forEach(TableCacheMetrics::unregister);
```

No need for stream or using `entrySet()`.

Reply:
Thanks. I have made it simpler as your suggested.